### DISCUSSION TITLE
Invalid `prisma.post.create()` invocation:

### DISCUSSION THREAD

**Discussion by rajanshresth** [prisma/prisma#19508] [Q&A]
This is my prisma schema file
![image](https://github.com/prisma/prisma/assets/76089000/04adde09-48dc-4145-9cd4-467b9f7dd8b2)
And, I've created api end point for POST
![image](https://github.com/prisma/prisma/assets/76089000/03ef9dab-b3f3-40ae-8504-58ee358f59c5)
I expect it to be working, instead i get:
![image](https://github.com/prisma/prisma/assets/76089000/b9e90182-ee0e-47f2-bc7a-692fc228ad41)

Environment & setup
OS: Ubuntu 22.04.2 LTS
Database:PlanetScale,
Node.js version: 20.2.0

Prisma Version
prisma version: 4.14.1

**Comment by ludralph** [comment]
This is a duplicate of this discussion https://github.com/prisma/prisma/discussions/19505. I'll be closing this in favour of the linked discussion.
